<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createNNWindow</class>
 <widget class="QDialog" name="createNNWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>101</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="sigmoid_rb">
      <property name="text">
       <string>Sigmoid</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="relu_rb">
      <property name="text">
       <string>RELU</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="tanh_rb">
      <property name="text">
       <string>tanh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="layerList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="neuronsSpinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>42</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Neurons: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Layers:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Activation Functions: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>createNNWindow</receiver>
   <slot>addLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>createNNWindow</receiver>
   <slot>removeLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layerList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>createNNWindow</receiver>
   <slot>selectItem(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>neuronsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>createNNWindow</receiver>
   <slot>setNeuronCount(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>released()</signal>
   <receiver>createNNWindow</receiver>
   <slot>createNN()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addLayer()</slot>
  <slot>slot2()</slot>
  <slot>removeLayer()</slot>
  <slot>slot3()</slot>
  <slot>selectItem(int)</slot>
  <slot>setNeuronCount(int)</slot>
  <slot>createNN()</slot>
 </slots>
</ui>
